hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/TokenGetNftInfoHandler.java

```diff
@@ -45,7 +49,7 @@ public TokenGetNftInfoHandler() {
     @Override
     public QueryHeader extractHeader(@NonNull final Query query) {
         requireNonNull(query);
-        return query.tokenGetInfoOrThrow().header();
+        return query.tokenGetNftInfoOrThrow().header();
```

Comment:
Seems wrong file is modified . It should be `TokenGetNftInfosHandler`

Reply:
I will fix it

Reply:
that specific change is correct by the way no, @Neeharika-Sompalli. It is supposed to be `tokenGetNftInfoOrThrow()`